src/main/java/wromaciej/moviesrental/movies/model/Actor.java

```diff
@@ -1,38 +1,57 @@
 package wromaciej.moviesrental.movies.model;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Collection;
 
 @Entity
 @Table(name = "actor", schema = "public")
 public class Actor {
 
     @Id
     @Column(name = "actor_id")
-    private Integer actorId;
+    private Integer id;
 
     @Column(name = "first_name")
     private String firstName;
 
     @Column(name = "last_name")
     private String lastName;
 
-    public Actor(){}
+    @ManyToMany(cascade = {CascadeType.ALL})
```

Comment:
CascadeType.ALL means during deletion of single Actor entity hibernate will delete all associated Movies (not Actor<->Movie mapping only). Take a look: https://vladmihalcea.com/a-beginners-guide-to-jpa-and-hibernate-cascade-types/ and consider which cascade option should be best there.

Reply:
done, I also removed DETACH type

Reply:
I think best would be placing all cascade types apart from DELETE